src/monitor.rs

```diff
@@ -145,7 +145,7 @@ async fn process_output(
     let client_options_ = client_options.clone();
 
     if address_wrapper == latest_address {
-        account_handle.generate_address_internal(&mut account).await?;
+        let _ = account_handle.generate_address_internal(&mut account).await;
```

Comment:
Why is the error handling removed? Maybe a comment would be nice.

Reply:
you're right, done